src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -284,7 +290,7 @@ public void run() {
                         BkBasicTest.POST,
                         BkBasicTest.HOST,
                         "",
-                        "Hello World!"
+                        BkBasicTest.HELLO_WORLD
```

Comment:
@dalifreire It looks like we can change this string to something else that is specific to this test